### PR TITLE
NIRO IS A GOOD TEACHER

### PR DESCRIPTION
**Pull request by taranraina**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -1,8 +1,14 @@
 #include <iostream>
 #include <string>
 #include "hello_world.h"
+#include "taran_add.h"
 
 int main()
 {
 	print_hello_world();
+
+	int x=4;
```
Comment (niroigen) [inline main.cpp]: I would prefer if you have spaces in between your variables

File: main.cpp [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+	int y=2;
```
Comment (niroigen) [inline main.cpp]: Same here

File: makefile [hunk 3]
```diff
@@ -9,12 +9,14 @@ CXXFLAGS = -Wall -g
 # ****************************************************
 # Targets needed to bring the executable up to date
 
-main: main.o hello_world.o
-	$(CXX) $(CXXFLAGS) -o main main.o hello_world.o
+main: main.o hello_world.o taran_add.o
```
Comment (niroigen) [inline makefile]: :+1:

File: makefile [hunk 4]
```diff
@@ -9,12 +9,14 @@ CXXFLAGS = -Wall -g
...
+	$(CXX) $(CXXFLAGS) -o main main.o hello_world.o taran_add.o
```
Comment (niroigen) [inline makefile]: :+1:

File: makefile [hunk 5]
```diff
@@ -9,12 +9,14 @@ CXXFLAGS = -Wall -g
...
 # The main.o target can be written more simply
 
-main.o: main.cpp hello_world.h hello_world.cpp
+main.o: main.cpp hello_world.h hello_world.cpp taran_add.cpp 
 	$(CXX) $(CXXFLAGS) -c main.cpp
 
-hellow_world.o: hello_world.h
+hellow_world.o: hello_world.h 
+
+taran_add.o: taran_add.h
```
Comment (niroigen) [inline makefile]: There should be an end of line for this file, please fix

File: taran_add.h [hunk 6]
```diff
@@ -0,0 +1,6 @@
+#ifndef TARAN_ADD_H
+#define TARAN_ADD_H
+
+int sum (int x, int y); 
+
+#endif
```
Comment (niroigen) [inline taran_add.h]: End of file missing here as well :smile:

File: main.cpp (hunk 1 above)
Comment (taranraina) [reply main.cpp]: i prefer not

File: main.cpp [hunk 7]
```diff
@@ -1,8 +1,14 @@
...
 #include <string>
 #include "hello_world.h"
+#include "taran_add.h"
 
 int main()
 {
 	print_hello_world();
+
+	int x = 4;
+	int y = 2;
```
Comment (taranraina) [inline main.cpp]: Hi Niro, changed the variables

File: makefile (hunk 5 above)
Comment (taranraina) [reply makefile]: done

**Comment by niroigen** [comment]
AWESOMEEEE

**Pull request merged**